### DISCUSSION TITLE
new UI for GitHub is vey bad

### DISCUSSION THREAD

**Discussion by 0xGhada** [community/community#58991] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/115355629/25b6a27a-f5b8-4db8-8aca-87d40278942e)

**Comment by Silent-Watcher** [comment]
I agree :(

**Comment by meppu** [comment, marked as answer]
If you meant the top bar, you should be able to close it from feature preview.
